packages/arb-token-bridge-ui/tailwind.config.js

```diff
@@ -53,7 +57,7 @@ module.exports = {
     extend: {
       display: ['group-hover'],
       textColor: ['disabled'],
-      backgroundColor: ['active', 'disabled'],
+      backgroundColor: ['active', 'disabled', 'even'],
```

Comment:
we're on tailwind v3 now so we can delete all the variants
don't need to add this one

feel free to leave it here though
it can be cleaned up with a tailwind config PR

https://tailwindcss.com/docs/upgrade-guide#remove-variant-configuration

Reply:
Sweet! Created an issue.
https://github.com/OffchainLabs/arb-token-bridge/issues/597